### PR TITLE
Feature/modularize

### PR DESCRIPTION
**Pull request by tvdw2000**

### CODE REVIEW

File: application/build.gradle [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+dependencies {
+    implementation project(':core')
+    implementation project(':adapters:outgoing:jpa')
+    implementation project(':adapters:incoming:rest')
+
+    // Spring
+    implementation 'org.springframework.boot:spring-boot-starter'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
```
Comment (bavodaniels) [inline application/build.gradle:37]: application shouldnt need starter web

and shouldnt need most of the dependencies below here
(javax, oauth2 client, AD, security)

**Comment by bavodaniels** [comment]
configureCors in application module should reside in rest adapter module

File: application/src/test/java/com/seatapp/controllers/LoginControllerTest.java [hunk 2]
```diff
@@ -15,7 +15,6 @@
 import org.springframework.web.context.WebApplicationContext;
 
 import javax.transaction.Transactional;
```
Comment (bavodaniels) [inline application/src/test/java/com/seatapp/controllers/LoginControllerTest.java:17]: tests should be in the module which defines them

**Comment by bavodaniels** [comment]
core/src/main/java/com/seatapp/domain/usermanagement/JwtUserDetails.java should be in rest module, 
and in the rest module you can block/allow certain calls to the core with the info from the user

and you can pass a POJO user object to the core (without any annotations or only with jpa annotations if that's your choice)

**Review by LanderTuyteleersXplore — CHANGES_REQUESTED** [review]
The core build.gradle file still contains dependencies to spring-boot-web & spring-boot oath. These should be removed.

**Review by bavodaniels — APPROVED** [review]

**Review by LanderTuyteleersXplore — APPROVED** [review]

**Pull request merged**